sycl/unittests/kernel-and-program/DeviceInfo.cpp

```diff
@@ -168,6 +169,9 @@ TEST_F(DeviceInfoTest, GetDeviceFreeMemory) {
 
   EXPECT_EQ(sizeof(FreeMemory), sizeof(uint64_t))
       << "Expect free_memory to be of uint64_t size";
+
+  EXPECT_GE(TotalMemory, FreeMemory)
+      << "Expect total_memory >= free_memory";
```

Comment:
How is this related to your changes? I am not against it, but it seems like something that should have its own patch.

Reply:
This test is not related. I will move this into a separate patch.